### ISSUE TITLE
Stopped Working After the Update to 2.4.0

### ISSUE THREAD

**Issue by svimes** [mozilla/testpilot-containers#674]
Containers stopped working on my Firefox ESR 52.2.1 (32-bit) on Windows after the update of the plugin to Containers Experiment 2.4.0. The plugin is enabled, but the context menu is not showing.

**Comment by jonathanKingston** [comment]
Are you on Windows Nightly by any chance?
Could you paste the contents of about:support please?

**Comment by svimes** [comment]
I don't think that I am running a nightly build. Here's my about::support:
[AboutSupport.zip](https://github.com/mozilla/testpilot-containers/files/1149366/AboutSupport.zip)
Thank you for checking!

**Comment by svimes** [comment]
My description wasn't clear enough - the issue is that the context menu that appears when you hover the mouse over the new tab button doesn't show:
https://testpilot.firefox.com/static/images/experiments/containers/details/detail1.e10ab74e.jpg

**Comment by groovecoder** [comment]
The UI behavior was changed from hover to long-press. Can you try to press-and-hold the "+" new tab button?

**Comment by groovecoder** [comment]
Also, when you installed the add-on (or got the update), it should have shown a "NEW" label on the browser action icon, and the panel should have explained the new long-press behavior. Do you remember seeing that?

**Comment by svimes** [comment]
Thank you, you're right about both:
- long press works - there's new popup menu (that looks less appealing than the previous one :( );
- there was a new icon, but when I pressed it, nothing happened.
Pressing the containers icon in the right doesn't work.

**Comment by jonathanKingston** [comment]
@groovecoder a few people have reported not seeing that screen which is a little odd. I tested that code pretty hard. Is it worth having a double check for the shield study?

**Comment by svimes** [comment]
![containerspressed](https://user-images.githubusercontent.com/13842681/28226584-b1960846-68a4-11e7-87ef-4428ea21c907.png)

**Comment by jonathanKingston** [comment]
Wait are you on Windows Nightly?

**Comment by svimes** [comment]
No, I am not, but I don't have control over the installation - it's managed by the systems administrators in my company.

**Comment by jonathanKingston** [comment]
Ah wait sorry just saw Firefox ESR 52.2.1. Was that even working pre the latest release? It shouldn't have been.
Unfortunately we no longer support 52 due to the migration to WebExtensions. It will work on the next ESR (however that is a long way off!). I don't think there is a way to get that working properly at this point unfortunately.

The longpress might continue to work which will allow you to create containers. Are you able to change about:config preferences?

**Comment by svimes** [comment]
It was working fine prior latest release. I can change the preferences in about:config

**Comment by jonathanKingston** [comment]
@svimes yeah we are migrating more and more to WebExtensions, some of which are not available in the ESR unfortunately. Backporting patches to ESR isn't a huge priority due to the numbers however it might be something we could do later because of the Tor population.

Anyway... I would advise ESR users uninstalling the extension through test pilot and using nightly. In about:config change the following after uninstall:
- privacy.userContext.enabled = true
- privacy.userContext.ui.enabled = true
- privacy.userContext.longPressBehavior = 2

It is likely this is how they currently are with the extension set up. To modify containers you can go to about:preferences#containers which is navigatable via the Privacy panel.
Basically this is a less feature rich version of what we have here but will keep working for all of the ESR period.

For the time being I am going to close this issue as we have already agreed on dropping support. I'm sorry about this I know it's not the best position.
